adcc/backends/test_psi4.py

```diff
@@ -111,9 +111,18 @@ def template_rhf_h2o(self, basis):
 
         # Test dipole
         mints = psi4.core.MintsHelper(wfn.basisset())
-        ao_dip = [np.array(comp) for comp in mints.ao_dipole()]
+        ao_dip = [-1.0 * np.array(comp) for comp in mints.ao_dipole()]
         operator_import_test(wfn, ao_dip)
 
+        # TODO: generalize, code block repeated below...
```

Comment:
Yes please. Could you make an attempt? It's always better to do this before we have to maintain the code in master.

Reply:
Done.